scripts/start_api

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+mkdir -p ./api
+git clone https://github.com/vinyldns/vinyldns ./api/
+./api/bin/docker-up-vinyldns.sh
```

Comment:
I _believe_ this starts both the portal and the API. You may want to consider `bin/docker-up-vinyldns.sh --api-only`, which will _only_ start the API.

Reply:
I will add this later. Wanted to do the bare minimum to get this working and explore the options a little. I've also been using the portal locally to see what test data is being generated used.